content/05.proteolysis.md

```diff
@@ -23,25 +23,26 @@ The enzyme choice mostly depends on the application.
 In general, for a mere protein identification mostly trypsin is the choice due to the reasons aforementioned.
 However, alternative enzymes can facilitate _de novo_ assembly when the genomic data information is limited in the public database repositories [@pmid:31615963; @pmid:30622160; @pmid:29990557; @doi:10.1016/j.actatropica.2022.106324; @DOI:10.1021/pr400173d].
 Use of multiple proteases for proteome digestion also can improve the sensitivity and accuracy of protein quantification [@PMID:30336047].
+Moreover, by providing an increased peptide diversity, the use of multiple proteases can expand sequence coverage and increase the probability of finding peptides which are unique to single proteins [@DOI: 10.1021/acs.jproteome.9b00330]. A multi-protease approach can also improve the identification of mispredicted N-Termini and signal peptides for small proteins [@DOI: 10.1021/acs.jproteome.1c00115]. Therefore, integrating multiple-protease data can increase the number of proteins identified [@DOI: 10.3390/ijms20225630; @DOI: 10.1074/mcp.M113.035170], the number of identified post-translational modifications [@DOI: 10.1021/acs.jproteome.9b00330; @DOI: 10.1016/j.celrep.2015.05.029] and decrease the ambiguity of the protein group list [@DOI: 10.1021/acs.jproteome.9b00330].
```

Comment:
```suggestion
Moreover, by providing an increased peptide diversity, the use of multiple proteases can expand sequence coverage and increase the probability of finding peptides which are unique to single proteins [@DOI:10.1021/acs.jproteome.9b00330; @DOI:10.1074/mcp.M113.034710; @DOI:10.1155/2014/960902]. 
A multi-protease approach can also improve the identification of N-Termini and signal peptides for small proteins [@DOI:10.1021/acs.jproteome.1c00115]. 
Overall, integrating multiple-protease data can increase the number of proteins identified [@DOI:10.3390/ijms20225630; @DOI:10.1074/mcp.M113.035170], the number of identified post-translational modifications detected [@DOI:10.1021/acs.jproteome.9b00330; @DOI:10.1016/j.celrep.2015.05.029; @DOI:10.1074/mcp.M113.034710] and decrease the ambiguity of the protein group list [@DOI:10.1021/acs.jproteome.9b00330].
```

Reply:
I agree, should I click on commit suggestion?

Reply:
Yes if you agree with the suggestions you can click commit, otherwise you can discuss them in these comments, or you can go back and change them to something different by working on your branch